docs/guides/cheatsheet/repl.rst

```diff
@@ -11,14 +11,20 @@ a semicolon and press "ENTER".
     db> select 5;
     {5}
 
-Alternatively, you can run the query without a semicolon with ⌥-Enter on macOS
+Alternatively, you can run the query without a semicolon with esc+return on macOS
 or Alt-Enter on Windows/Linux.
 
 .. code-block:: edgeql-repl
 
     db> select 5
     {5}
 
+.. tip::
+
+    This esc+return or Alt-Enter trick is also very useful when your cursor is in
+    the middle of a long query and you want to run it directly without moving the
+    cursor to the end of the query.
```

Comment:
```suggestion
    The Alt-Enter / ESC+Return trick is also very useful when your cursor is in
    the middle of a long query and you want to run it directly without moving the
    cursor to the end of the query.
    
    Terminals like iTerm allow to configure left Option key to act as ESC+, which
    makes Alt-Enter also work on macOS.
```

Reply:
Let's mention the "Use Option as Meta option" here for Terminal.app users

Reply:
> Terminals like iTerm allow to configure

Edit: "Terminals like iTerm allow *you* to configure *the* left Option"

Reply:
Good catch and good suggestion on Terminal.app. I've updated with some detailed instructions for both iTerm and Terminal.app